PINRemoteImage.podspec

```diff
@@ -19,12 +19,22 @@ Pod::Spec.new do |s|
 
   s.platform     = :ios, '6.0'
   s.requires_arc = true
+  
+  # Exclude optional FLAnimatedImage module
+  s.default_subspec = 'Core'
```

Comment:
@Kurry I think it better that the default option be to include the pod? Everyone who is using it now is assuming that FLAnimatedImage is available.

Is it difficult to turn off if the default option is to include it? I'm not familiar enough with subspecs…